### ISSUE TITLE
Not able to create transaction from Isolated Module

### ISSUE THREAD

**Issue by AbhishekJadhavStackera** [airgap-it/airgap-coin-lib#69]
I'm creating an Isolated Module with the identifier `iso_eth`. When I try to make a send transaction, it shows the below error from the wallet app. I used Logcat to get the error log.
```text
Error: SERIALIZER(SCHEMA_DOES_NOT_EXISTS): Schema 5-iso_eth does not exist
```

**Comment by AndreasGassmann** [comment]
Hi. Sorry for the late reply. Is your source code public so we could take a look? And did you take a look at the example isolated module and follow its setup?

**Comment by AbhishekJadhavStackera** [comment]
Yeah, I followed exactly how it is mentioned in the docs to create an Isolated Module. Actually, I've already created an Isolated Module for BSC and it's working fine. Now I'm looking to extend the token list in the current default Ethereum module. So, I just took the present module code and modified the identifier and token list according to my needs, keeping the scripts the same as those I used while building the BSC module.
